ComponentKit/Utilities/CKSectionedArrayController.mm

```diff
@@ -130,6 +132,8 @@ - (void)enumerateObjectsInSectionAtIndex:(NSInteger)sectionIndex usingBlock:(CKS
 
 - (CKArrayControllerOutputChangeset)applyChangeset:(CKArrayControllerInputChangeset)changeset
 {
+  verifyChangeset(changeset, _sections);
```

Comment:
This only applies to `CKComponentDataSource` right? Do we want to support `CKTransactionalDataSource` as well?